### PR TITLE
Adding calling the ComputerName parameter to the example

### PR DESCRIPTION
**Pull request by mbreakey3**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/configurations.md](https://github.com/PowerShell/PowerShell-Docs/blob/mbreakey3-patch-1/dsc/configurations.md) | :warning:Warning | | [Details](#user-content-dsc%2Fconfigurations.md)

<a name="dsc%2Fconfigurations.md"></a>
### [dsc/configurations.md](https://github.com/PowerShell/PowerShell-Docs/blob/mbreakey3-patch-1/dsc/configurations.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C18%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201712182314094120-1964%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jROaLsfFP1Z6sKMGw5LpvF3TytUCwG0QE%2FY7%2BGpPmkY%3D&st=2017-12-18T23%3A51%3A56Z&se=2018-01-18T23%3A56%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: dsc/configurations.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Configuration MyDscConfiguration {
 		}
 	}
 }
-MyDscConfiguration
+MyDscConfiguration -ComputerName "MyComputer"
```
Comment (SteveL-MSFT) [inline dsc/configurations.md]: I think it's better to use this syntax:

```powershell
MyDscConfiguration -ComputerName <MyComputer>
```

Comment (joeyaiello) [reply dsc/configurations.md]: Even inside a comment-block example? I don't agree, this should be copy/paste, change your values.

Comment (SteveL-MSFT) [reply dsc/configurations.md]: The quotes to me imply they are needed.  The angle brackets is pretty universally accepted as meaning a required parameter value.

Comment (mbreakey3) [reply dsc/configurations.md]: Done

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/configurations.md](https://github.com/PowerShell/PowerShell-Docs/blob/mbreakey3-patch-1/dsc/configurations.md) | :warning:Warning | | [Details](#user-content-dsc%2Fconfigurations.md)

<a name="dsc%2Fconfigurations.md"></a>
### [dsc/configurations.md](https://github.com/PowerShell/PowerShell-Docs/blob/mbreakey3-patch-1/dsc/configurations.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C3%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801031632472082-1964%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yUDWWYAZtirF%2FZmqGSglizgxMMuRDGTRzA3eueWGNn8%3D&st=2018-01-03T17%3A02%3A41Z&se=2018-02-03T17%3A07%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**